src/manage_sql.c

```diff
@@ -9297,7 +9297,7 @@ alert_script_exec (const char *alert_id, const char *command_args,
 
   script = g_build_filename (script_dir, "alert", NULL);
 
-  if (!g_file_test (script, G_FILE_TEST_EXISTS))
+  if (! gvm_file_is_readable (script))
```

Comment:
Not sure about the space, everywhere else uses !call.

Reply:
Changed this in 2ffceab.